nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -132,15 +139,17 @@ else
     system_root=$closure
     # Create a temporary file ending in .closure (so nixos-prepare-root knows to --import it) to transport the store closure
     # to the filesytem we're preparing. Also delete it on exit!
-    nix-store --export $(nix-store -qR $closure) > $system_closure
+    # Background, so that we don't block while trying to write to the FIFO
+    # $system_closure refers to
```

Comment:
only a small nitpick:

```diff
- # Background, so that we don't block while trying to write to the FIFO
+ # Run in background to not block while trying to write to the FIFO
  # $system_closure refers to